test/testdrive/introspection-sources.td

```diff
@@ -0,0 +1,74 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
There is already `test/testdrive/logging.td` that also tests introspection sources. Does it make sense to merge these two?

Reply:
The comment at the top of logging.td is what caught my eyes:
```
# This test only verifies that the log relations are published, not that they
# have any specific output.
```
Since the new tests were going to actually check the specific output in the introspection source, I thought it was better to keep the tests separate. Another advantage of the separation is that `introspection_sources.td` depends on `testdrive` retries, while I believe that `logging.td` does not currently.